dropshot_endpoint/src/lib.rs

```diff
@@ -173,7 +173,8 @@ fn do_channel(
         ChannelProtocol::WEBSOCKETS => {
             // here we construct a wrapper function and mutate the arguments a bit
```

Comment:
```suggestion
            // Here we construct a wrapper function and mutate the arguments a bit
```

Reply:
Done in fb3183179de98c070dab4dc17be9a177f58e64d3.